src/components/global/FkH1.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import styled from "styled-components"
+
+const FkH1 = styled.h1`
```

Comment:
¿qué es `FkH1`? Preferiría que el nombre sea completo para que sea más fácil de entender

Reply:
Follow https://github.com/FoodKlore/Landing/pull/4#discussion_r366637505 , y decime que te parece la idea, I'm down to changing it to a less complex name.